tests/ref_jobs/fit_curve_odc_ref.py

```diff
@@ -17,9 +17,9 @@
 _1_19 = oeop.clip(**{'x': _23_20, 'min': 0, 'max': 4000})
 _22_18 = oeop.apply(**{'process': _1_19, 'data': _1_19, 'context': ''})
 _wz26aglyi_5 = oeop.multiply(**{'x': _p42lrxmbq_16, 'y': _p42lrxmbq_16})
-_kryhimf6r_6 = oeop.array_element(**{'data': [1, 1, 1], 'index': 0})
-_jxs4umqsh_10 = oeop.array_element(**{'data': [1, 1, 1], 'index': 1})
-_8jjjztmya_12 = oeop.array_element(**{'data': [1, 1, 1], 'index': 2})
+_kryhimf6r_6 = oeop.array_element(**{'data': _parameters, 'index': 0})
+_jxs4umqsh_10 = oeop.array_element(**{'data': _parameters, 'index': 1})
+_8jjjztmya_12 = oeop.array_element(**{'data': _parameters, 'index': 2})
```

Comment:
I'm afraid there still is an error somewhere... `_parameters` is not "node" which would correspond to a Python variable here. I think the `from_parameter` key is not resolved correctly. Looking at the process graph the original [1, 1, 1] would be the correct result.

Reply:
Thank you, that's really good to know! My first solution was `'data' : [1, 1, 1]` but this was the part where I needed to check if my `from_parameter` is a string or anything else. In all other cases where `from_parameter` is used it points to a `str` but this is the only case where it points to a `list`.